doxia-modules/doxia-module-xhtml5/src/main/java/org/apache/maven/doxia/module/xhtml5/Xhtml5Parser.java

```diff
@@ -158,7 +158,7 @@ protected void handleEndTag(XmlPullParser parser, Sink sink)
         } else if (parser.getName().equals(TITLE.toString())) {
             sink.title_();
         } else if (parser.getName().equals(BODY.toString())) {
-            consecutiveSections(0, sink, null);
+            emitHeadingSections(0, sink, false);
```

Comment:
This one confuses me

Reply:
This is just about replacing the deprecated method with the non deprecated one.

Reply:
Alright.